src/schema/posts.ts

```diff
@@ -1250,6 +1250,7 @@ export const resolvers: IResolvers<any, Context> = {
           throw new ValidationError('URL is not valid');
         }
 
+        const strippedCommentary = commentary ? commentary.trim() : '';
```

Comment:
Should we probably keep the `null` value if it is empty? WDYT? @idoshamun

Reply:
Yes, let's set it to null

Reply:
Maybe it makes more sense to even put this `commentary ? commentary.trim() : null`` inside the `createSharePost` function. This way we have it only in one place

Reply:
Good idea, reduce the duplicate code. This change has been done now

Reply:
I noticed the change, but we are now missing the `trim` part of the commentary when there is an actual value. Ido's suggestion would probably be the best route to go about it.

`commentary ? commentary.trim() : null`